aws/resource_aws_dx_gateway.go

```diff
@@ -0,0 +1,124 @@
+package aws
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/directconnect"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsDxGateway() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsDxGatewayCreate,
+		Read:   resourceAwsDxGatewayRead,
+		Delete: resourceAwsDxGatewayDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
If `terraform import` is supported please note in the documentation and add an import acceptance test.
I did that for the existing Direct Connect resources in https://github.com/terraform-providers/terraform-provider-aws/pull/2992.